pact/base.py

```diff
@@ -96,15 +97,21 @@ def on_timeout(self, callback, *args, **kwargs):
         self._timeout_callbacks.append(functools.partial(callback, *args, **kwargs))
         return self
 
+    def set_timeout(self, timeout_seconds):
```

Comment:
Also please add an explanation about this in http://pact.readthedocs.io/en/stable/user_guide.html#waiting-on-pacts